.env-example

```diff
@@ -1 +1,21 @@
-PGDATABASE=database_name_here
\ No newline at end of file
+PGDATABASE=database_name_here
+

```

Comment:
These instructions are no doubt very useful for your own reference, but I wouldn't keep them in this .env-example file as they aren't really related to .env instructions and useage.

Maybe create a file called notes.txt and store stuff like this there instead?